### DISCUSSION TITLE
Through-hole Tulip

### DISCUSSION THREAD

**Discussion by Jayson330** [shorepine/tulipcc#130] [Show and tell]
Hey folks,

I'm working on the through-hole Tulip this weekend and before I solder the sound card on, I want to make absolutely sure that this is the correct pin placement. 

Thanks! 

![IMG20230729152228](https://github.com/bwhitman/tulipcc/assets/53438355/e5c18bc4-35d3-4446-87c2-72997afc99c4)

**Comment by bwhitman** [comment]
Nice! That looks right to me. BCLK is the pin closest to the edge of the board. 
<img width="1174" alt="image" src="https://github.com/bwhitman/tulipcc/assets/76612/a2bf69ad-c410-4d13-84f9-374514e9a489">

**Reply by Jayson330** [reply]
Thanks for the quick response!
